otherlibs/unix/unix.mli

```diff
@@ -217,7 +217,9 @@ type wait_flag =
 
 val execv : string -> string array -> 'a
 (** [execv prog args] execute the program in file [prog], with
-   the arguments [args], and the current process environment.
+   the arguments [args], and the current process environment.  
+   Note that the first argument is by convention the filename of
```

Comment:
first argument is slightly ambiguous: it may refer to `prog`. I would propose to be very explicit! `the first argument, [args.(0)], `